src/ansi.rs

```diff
@@ -812,6 +775,8 @@ pub enum Mode {
     SwapScreenAndSetRestoreCursor = 1049,
     /// ?2004
     BracketedPaste = 2004,
+    /// Synchronized update start/end.
+    SyncUpdate = 2026,
```

Comment:
This forces Alacritty to handle this mode in our `set_mode`/`unset_mode` match statement. But I didn't want to handle this mode completely separately, so it's probably fine?

Reply:
The thing is that it's confusing for the consumers, because this mode doesn't tell a thing.

Though, we can easily check for it separately. It just strange that we have a mode that can never be populated to downstream and that they could think that they need to freeze the grid or something like that.

Reply:
Yeah that's fair.